scripts/lib/api/processHtml.test.ts

```diff
@@ -247,6 +249,26 @@ test("replaceSourceLinksWithGitHub()", () => {
   );
 });
 
+test("convertRubricsToHeaders()", () => {
+  const doc = Doc.load(`<p class="rubric">Example</p>
+    <p class="rubric">Examples</p>
+    <p class="rubric">References</p>
+    <p class="rubric">Reference</p>
+    <p class="rubric">Attributes</p>
+    <p class="rubric">Methods</p>
+    <p class="rubric">Simple examples</p>
+    <p class="rubric">Example code</p>`);
+  convertRubricsToHeaders(doc.$, doc.$main);
+  doc.expectHtml(`<strong>Example</strong>
+    <strong>Examples</strong>
+    <strong>References</strong>
+    <h2>Reference</h2>
+    <h2>Attributes</h2>
+    <h2>Methods</h2>
+    <h2>Simple examples</h2>
+    <h2>Example code</h2>`);
```

Comment:
These should be `<strong>` for the same reason as Reference.

Reply:
What should we match then? Anything containing example? Or should we go the other route and have everything be `strong`, with `Methods` and `Attributes` being the exceptions?

Reply:
> should we [...] have everything be `strong`, with `Methods` and `Attributes` being the exceptions?

With another look at the docs, I think that might be best. All the other rubrics I can see should be `strong` IMO.

<details>
 <summary>Other rubrics</summary>

```html
python/sources/qiskit/0.45.2/artifact/apidoc/pulse.html
2120:<p class="rubric">Notes</p>
2178:<p class="rubric">Notes</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.pulse.ScheduleBlock.html
53:<p class="rubric">Pulse Builder</p>
57:<p class="rubric">Alignment contexts</p>
68:<p class="rubric">Nested blocks</p>
75:<p class="rubric">Unsupported operations</p>
83:<p class="rubric">Subroutine</p>
95:<p class="rubric">Program Scoping</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.pulse.Schedule.html
116:<p class="rubric">Notes</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.transpiler.passes.ConsolidateBlocks.html
51:<p class="rubric">Notes</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.pulse.library.SymbolicPulse.html
66:<p class="rubric" id="symbolic-pulse-envelope">Envelope function</p>
82:<p class="rubric" id="symbolic-pulse-constraints">Constraint functions</p>
92:<p class="rubric" id="symbolic-pulse-eval-condition">Maximum amplitude validation</p>
169:<p class="rubric" id="symbolic-pulse-serialize">Serialization</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.result.QuasiDistribution.html
184:<p class="rubric">Notes</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.quantum_info.SparsePauliOp.html
58:<p class="rubric">Data type of coefficients</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.circuit.QuantumCircuit.html
1291:<p class="rubric">Notes</p>
1522:<p class="rubric">Notes</p>
2516:<p class="rubric">Notes</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.transpiler.passes.StochasticSwap.html
49:<p class="rubric">Notes</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.transpiler.passes.ConstrainedReschedule.html
69:<p class="rubric">Notes</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.algorithms.optimizers.NFT.html
64:<p class="rubric">Notes</p>

python/sources/qiskit/0.45.2/artifact/stubs/qiskit.transpiler.passes.AlignMeasures.html
91:<p class="rubric">Notes</p>
```
</details>

Reply:
100% agreed.

Reply:
Done in 419645b5d7356c00d53fb26059186b98b2656690